Formula/s/sox.rb

```diff
@@ -47,8 +47,9 @@ class Sox < Formula
   end
 
   def install
-    args = std_configure_args
+    ENV.append_to_cflags "-Wno-incompatible-function-pointer-types" if DevelopmentTools.clang_build_version >= 1500
```

Comment:
```suggestion
    ENV.append_to_cflags "-Wno-incompatible-function-pointer-types" if DevelopmentTools.clang_build_version >= 1600
```
?

Reply:
I don't know why it didn't trigger, but all other uses of `-Wno-incompatible-function-pointer-types` trigger on clang 15 as well, so I'm keeping things consistent.